### DISCUSSION TITLE
Need a simple menu for displaying models.

### DISCUSSION THREAD

**Discussion by WentsWorths** [google/model-viewer#4423] [Q&A] [answered]
Hello im making a project and need to figure out how to display different models on the same page when a button is clicked.
Specifically, the page loads with its current model, but when a menu button is clicked the model updates to a different one without changing to a new page.

 https://yzy.vercel.app/style (reference example)

<img width="1274" alt="Screen Shot 2023-08-13 at 10 48 26 AM" src="https://github.com/google/model-viewer/assets/142162204/cdc0ba8d-1a7e-449c-ad92-825c029d8b71">

https://www.saveyourwork.net/app/   (Heres my example)

Where/what in my backend would I have to change so that if I clicked "SHADES" the current model would update to the new one?

**Comment by adrianposteuca** [comment]
Add an event listener "onClick" and change the MV.src ="./newmodel.glb"

**Reply by saveyourworkllc** [reply]
Hey, I got it to work, thank you so much!

**Comment by WadeMorrison** [comment, marked as answer]
To also pitch in, you can look at this example that's in the documentation here: 
[https://modelviewer.dev/examples/augmentedreality/#webXR](https://modelviewer.dev/examples/augmentedreality/#webXR)

And then tweak it to fit your settings.

**Reply by saveyourworkllc** [reply]
Helped out sooo much thank you!!
